wordpress-mobile/WordPress-iOS
# Xcode 13 support + iOS 15 fixes

Adds support to Xcode 13 (including CI) and fixes issues for iOS 15 + Xcode 13 combo.

Included issues:

* #17225
* #17227
* #17233
* #17379

### To test

1. Run the app
2. Make sure the app has the same look 'n feel
3. Make sure CI is 🟢 

## Regression Notes
1. Potential unintended areas of impact
The whole project

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual test

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
